resources/sass/components/forum/_post.scss

```diff
@@ -282,15 +317,15 @@
     cursor: pointer;
 }
 
-@media screen and (max-width: 576px) {
+@media screen and (max-width: 1358px) {
```

Comment:
With this change, sparkles on a user spread across the entire width of the page.